stdlib/builtins.pyi

```diff
@@ -109,19 +113,19 @@ class object:
     def __dir__(self) -> Iterable[str]: ...
     def __init_subclass__(cls) -> None: ...
 
-class staticmethod(object):  # Special, only valid as a decorator.
```

Comment:
Nit: These comments should be deleted. They seem misleading and mypy-specific to me.